pom.xml

```diff
@@ -196,33 +206,6 @@
         </exclusion>
       </exclusions>
     </dependency>
-    <dependency>
-      <groupId>org.apache.avro</groupId>
-      <artifactId>avro</artifactId>
-      <version>1.7.7</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.parquet</groupId>
-      <artifactId>parquet-avro</artifactId>
-      <version>1.8.1</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.apache.avro</groupId>
-          <artifactId>avro</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.avro</groupId>
-      <artifactId>avro-mapred</artifactId>
-      <classifier>hadoop2</classifier>
-      <version>1.7.7</version>
-    </dependency>
-    <dependency>
-      <groupId>xerces</groupId>
```

Comment:
Is this change intentional?

Reply:
yeah, it's moved into the format plugins in https://github.com/caskdata/hydrator-plugins/pull/860. This shouldn't be in the plugin at all, but is required to workaround https://issues.cask.co/browse/CDAP-14562